src/js/pouch.js

```diff
@@ -94,31 +95,17 @@ gpii.pouch.cleanup = function (that) {
 
     var promises = [];
     fluid.each(that.databaseInstances, function (db, key) {
-            // If we use the simpler method, the next attempt to recreate the database fails with a 409 document update conflict.
-            //var promise = db.destroy();
-
-            // Instead, we retrieve the list of all document IDs and revisions, and then bulk delete them.
-            var promise = db.allDocs()
-                .then(function (result) {
-                    var bulkPayloadDocs = fluid.transform(result.rows, gpii.pouch.transformRecord);
-                    var bulkPayload     = {docs: bulkPayloadDocs};
-                    return db.bulkDocs(bulkPayload);
-                })
-                .then(function () {
-                    fluid.log("Deleted existing data from database '" + key + "'...");
-                    return db.compact();
-                })
-                .then(function () {
-                    fluid.log("Compacted existing database '" + key + "'...");
-                })
-                .catch(fluid.fail); // jshint ignore:line
-
-            promises.push(promise);
-        }
-    );
+        var promise = db.destroy()
```

Comment:
The context here scared the hell out of me, I thought I had omitted the colon like some kind of beast.